### PR TITLE
Cleanup when passing custom text to OOH Pane

### PR DESCRIPTION
**Pull request by elopezanaya**
### Description
_Include a description of the problem to be solved_

when certain text is passed to be displated in the OOH pane, this may mess with UX experience

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

#### Passing xss 
image

#### Passing plain text
image

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/common/utils/xssUtils.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import DOMPurify from "dompurify";
+
+const xssPatterns = [
```
Comment (Erli-ms) [inline chat-widget/src/common/utils/xssUtils.ts:3]: this would not be a complete list of patterns, right? why cannot we always use the sanitized title?

Comment (elopezanaya) [reply chat-widget/src/common/utils/xssUtils.ts:3]: if the text is flagged as XSS, it will be a risk to use the text, is better to default to predefined text

File: chat-widget/src/components/ooohpanestateful/OOOHPaneStateful.tsx [hunk 2]
```diff
@@ -13,9 +12,12 @@ import { ITimer } from "../../common/interfaces/ITimer";
 import { OutOfOfficeHoursPane } from "@microsoft/omnichannel-chat-components";
 import { TelemetryHelper } from "../../common/telemetry/TelemetryHelper";
 import { defaultGeneralStyleProps } from "./common/defaultStyleProps/defaultgeneralOOOHPaneStyleProps";
+import { detectAndCleanXSS } from "../../common/utils/xssUtils";
 import useChatContextStore from "../../hooks/useChatContextStore";
 
-let uiTimer : ITimer;
+let uiTimer: ITimer;
+const FALLBACK_TITLE_TEXT = "Thanks for contacting us. You have reached us outside of our operating hours. An agent will respond when we open.";
```
Comment (Sonam-Ligade) [inline chat-widget/src/components/ooohpanestateful/OOOHPaneStateful.tsx:19]: can we refer this from Texts.OOOHPaneTitleText?

**Review by Sonam-Ligade — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]
